x-pack/plugins/security_solution/public/management/components/context_menu_with_router_support/context_menu_with_router_support.tsx

```diff
@@ -56,7 +63,20 @@ export const ContextMenuWithRouterSupport = memo<ContextMenuWithRouterSupportPro
           />
         );
       });
-    }, [handleCloseMenu, items]);
+    }, [handleCloseMenu, items, maxWidth]);
+
+    type AdditionalPanelProps = Partial<EuiContextMenuPanelProps & HTMLAttributes<HTMLDivElement>>;
+    const additionalContextMenuPanelProps = useMemo<AdditionalPanelProps>(() => {
```

Comment:
same here, wouldn't be better to put this into a styled component?

Reply:
yeah, I guess I could have the styled component check the prop and apply the style. At one point I read that we should keep usage of styled components to a minimum due to browser overhead (insert/remove/generate CSS stylesheets). Being that the styled component's stylesheet could get generated multiple times (for different widths), I just went with the Style attribute

Reply:
Got it, for some reason I thought we should avoid use style props and use styled components instead 😅